TODO.md

```diff
@@ -24,6 +24,41 @@
 - [x] Filter by kind
 - [x] Task detail view
 - [ ] Horizontal pan (including drag, keyboard, horizontal scroll wheel)
-- [ ] Keyboard bindings (e.g., arrow keys to select panels, space bar to toggle expand/collapse)
+- [ ] Keyboard bindings (e.g., arrow keys to select panels, space bar to toggle expand/collapse, ESC key to close popups)
 - [ ] Editable key bindings?
-- [ ] Better error handling (e.g., when the provided URL 404s, or parsing fails)
+- [ ] Better error handling (e.g., when the provided URL 404s, there's a permission issue, or parsing fails)
+- [ ] Make text in popup boxes copyable
+- [ ] Make highlighted boxes clearer
+
+  Currently a highlighted box is shown in red, but that color is already in use in the default color scheme, so highlighted items don't stand out
+
+  Possible solutions:
+
+  - "halo" effect for highlighted box
+  - different fill pattern
+  - different line color
+  - saturate all other boxes except the highlighted one (as happens for search)
```

Comment:
Indent to line up with the first line of the bullet, so it gets parsed properly as Markdown.

Reply:
Doing it this way

<img width="1481" alt="Screenshot 2023-09-27 at 5 31 11 PM" src="https://github.com/StanfordLegion/prof-viewer/assets/215728/c6918a54-4776-4b2f-8a1c-7158fc448326">

results in github misparsing it as a code block:

<img width="1481" alt="Screenshot 2023-09-27 at 5 31 23 PM" src="https://github.com/StanfordLegion/prof-viewer/assets/215728/710e89ec-1a6b-4730-9160-e05b5da092e3">

The original parses properly (at least by github):

<img width="1481" alt="Screenshot 2023-09-27 at 5 33 12 PM" src="https://github.com/StanfordLegion/prof-viewer/assets/215728/38eaf5ab-737b-4973-ae81-2a6b483972b3">